signer/tests/integration/emily.rs

```diff
@@ -297,10 +301,11 @@ async fn deposit_e2e() {
                 });
 
             // Return the deposit tx
-            client.expect_get_tx().returning(move |txid| {
-                let res = if *txid == deposit_tx.compute_txid() {
+            let deposit_tx_ = deposit_tx.clone();
+            client.expect_get_tx().once().returning(move |txid| {
```

Comment:
nit: spread call-chains over newlines (like you did below)

Reply:
rustfmt reverts it 😅

Reply:
Whaat :rofl: bad `rustfmt`....!